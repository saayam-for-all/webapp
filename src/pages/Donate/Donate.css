.donation-tab {
  background-color: #f5f5f5;
  padding: 20px;
  border-radius: 10px;
  box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.1);
  text-align: center;
  animation: fadeIn 1s ease;
}

.donation-input {
  margin: 20px 0;
  display: flex;
  align-items: center;
  justify-content: center;
  font-size: 24px;
}

.donation-input input {
  width: 60px;
  margin: 0 10px;
  font-size: 24px;
  border: none;
  outline: none;
}

button {
  background-color: #4caf50;
  color: white;
  border: none;
  padding: 10px 20px;
  font-size: 18px;
  border-radius: 5px;
  cursor: pointer;
  transition: background-color 0.3s ease;
}

button:hover {
  background-color: #45a049;
}

@keyframes fadeIn {
  from {
    opacity: 0;
  }

  to {
    opacity: 1;
  }
}
