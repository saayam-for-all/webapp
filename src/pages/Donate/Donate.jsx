import React, { useState } from "react";
import "./Donate.css";
import QRIMG from "../../assets/QR.png";

const Donation = () => {
  return (
    <div className="donation-tab mt-20">
      <h1 className="tect-center">You can scan or click on Donate </h1>
      <div className="flex items-center justify-center mt-6">
        <img src={QRIMG} alt="qr" />
      </div>
      <a
        href="https://www.paypal.com/donate/?hosted_button_id=4KLWNM5JWKJ4S"
        className="btn btn-accent mt-6"
        target="_blank"
      >
        Dontate
      </a>
    </div>
  );
};

export default Donation;
