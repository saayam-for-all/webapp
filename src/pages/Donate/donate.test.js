import { render } from "@testing-library/react";
import Donate from "./Donate";

describe("Donate", () => {
  it("renders correctly", () => {
    const tree = render(<Donate />);
    expect(tree).toMatchSnapshot();
  });
});
