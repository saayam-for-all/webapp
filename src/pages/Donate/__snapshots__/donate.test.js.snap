// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Donate renders correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="donation-tab mt-20"
      >
        <h1
          class="tect-center"
        >
          mockTranslate(SCAN_ON_DONATE)
        </h1>
        <div
          class="flex items-center justify-center mt-6"
        >
          <img
            alt="qr"
            src="QR.png"
          />
        </div>
        <a
          class="btn btn-accent mt-6"
          href="https://www.paypal.com/donate/?hosted_button_id=4KLWNM5JWKJ4S"
          target="_blank"
        >
          mockTranslate(DONATE)
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="donation-tab mt-20"
    >
      <h1
        class="tect-center"
      >
        mockTranslate(SCAN_ON_DONATE)
      </h1>
      <div
        class="flex items-center justify-center mt-6"
      >
        <img
          alt="qr"
          src="QR.png"
        />
      </div>
      <a
        class="btn btn-accent mt-6"
        href="https://www.paypal.com/donate/?hosted_button_id=4KLWNM5JWKJ4S"
        target="_blank"
      >
        mockTranslate(DONATE)
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
